src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -15,9 +12,12 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// This class implements Test-Json command.
     /// </summary>
-    [Cmdlet(VerbsDiagnostic.Test, "Json", HelpUri = "")]
+    [Cmdlet(VerbsDiagnostic.Test, "Json", DefaultParameterSetName = ParameterAttribute.AllParameterSets, HelpUri = "")]
     public class TestJsonCommand : PSCmdlet
     {
+        private const string SchemaPathParameterSet = "SchemaPath";
+        private const string SchemaStringParameterSet = "SchemaString";
+
         /// <summary>
         /// An JSON to be validated.
```

Comment:
I suggest changing "An JSON" to "A JSON string".

Reply:
Well spotted, thanks!

Reply:
Thanks for updating, @beatcracker , but my suggestion was also to qualify the term _JSON_ with the word _string_ or _text_, given the use of the indefinite article ("a" / "an"): JSON is a _notation_, whose _representation_ is (typically and sensibly) a _string_.

Therefore my suggestion is to use either of the following:

* Either: just "JSON", _without_ "a", relying on readers _inferring_ the string representation: "JSON to be validated" (or "JSON _text_ to be validated")

* Or, more explicitly: "JSON string", _with_ "a": "A JSON _string_ to be validated"

Reply:
@beatcracker - oops! Never mind - just saw the most recent push.

Reply:
Yeah, my bad should've pushed them together. Thanks for the detailed explanation.